variables.tf

```diff
@@ -141,3 +141,9 @@ variable "extra_ui_static_env_vars" {
   default     = {}
   description = "Additional environment variables for UI static app"
 }
+
+variable "with_public_ip" {
```

Comment:
@andreinechaev if you expose, you'll need to add to the README as well

Reply:
nit: now that I think of it we should probably make the description more descriptive too (I understand this PR copied it from the submodule, but still). Maybe something like "enable public IP assignment for the metadata service".